style.css

```diff
@@ -0,0 +1,84 @@
+
+*{
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+    body{
+      font-family: sans-serif;
+      letter-spacing: 1px;
+    }
+    nav{
+      display: flex;
+      flex-direction: row;
+      width: fit-content;
+      justify-content: space-between;
+      align-items: center;
+    }
+
+    nav ul{
+      list-style-type: none;
+      display: flex;
```

Comment:
Please remove unnecessary spaces in your code and add a single space between your selectors and the opening curly braces. This makes your work look neat.